src/drawers/areaDrawer.ts

```diff
@@ -16,7 +16,7 @@ export module Drawers {
     }
 
     public selectionForIndex(index: number) {
-      return this.renderArea().select(this.selector());
```

Comment:
Using the `renderArea()` getter does not reset the caching state (as it should).  So, I don't see why we shouldn't reduce the scope for d3 to search for the right selection